eclipse-collections-code-generator-maven-plugin/src/main/java/org/eclipse/collections/codegenerator/maven/GenerateMojo.java

```diff
@@ -31,12 +31,12 @@
     /**
      * Skips code generation if true.
      *
-     * @parameter expression="${skipCodeGen}"
+     * @parameter property="skipCodeGen"
      */
     private boolean skipCodeGen;
 
     /**
-     * @parameter expression="${project.build.directory}/generated-sources"
+     * @parameter default-value="${project.build.directory}/generated-sources"
```

Comment:
Thanks for the explanation. Makes sense to me. Do you mind rebasing on top of master?